src/docs/truffle-dashboard/truffle-dashboard-debugger.md

```diff
@@ -0,0 +1,74 @@
+
+When a transaction appears in Truffle Dashboard for signature confirmation, you’ll see a Simulate button to the right of the other options on the card. Clicking this button will launch the debugger for this transaction.
+
+![Debug a transaction from the Signature Confirmation screen by clicking the Debug button on a card](/img/docs/truffle-dashboard/truffle-dashboard-debugger/debug-button.png)
+
+## Debugging from a Transaction Hash
+
+You can paste any arbitrary transaction hash into the debugger rather than using the signature confirmation cards. Paste the desired transaction hash into the input at the upper-right of the debugger screen and click the debug button or press enter to start the debugger session. Note that the Debug button will be disabled until a valid transaction hash is provided.
+
+![Debug a transaction using its hash by pasting the hash into the text input on the Debugger screen](/img/docs/truffle-dashboard/truffle-dashboard-debugger/debug-hash.png)
+
+## The Debugger UI
+
+The debugger UI is composed of a top menu and 4 panels.
+
+![The Dashboard Debugger interface](/img/docs/truffle-dashboard/truffle-dashboard-debugger/debug-interface.png)
+
+The top menu contains 6 buttons in addition to the transaction hash input:
+
+- **Continue (Until Next Breakpoint)**: If a breakpoint is set, this continues execution until the next breakpoint is reached or the last line is executed. Note this will be inactive until you set a breakpoint.
+- **Step Next**: Steps to the next logical statement or expression in the source code. For example, evaluating sub expressions will need to occur first before the virtual machine can evaluate the full expression. Use this command if you'd like to analyze each logical item the virtual machine evaluates.
+- **Step Over**: Steps over the current line, relative to the position of the statement or expression currently being evaluated in the Solidity source file. Use this command if you don't want to step into a function call or contract creation on the current line, or if you'd like to quickly jump to a specific point in the source file.
+- **Step Into**: Steps into the function call or contract creation currently being evaluated. Use this command to jump into the function and quickly start debugging the code that exists there.
+- **Step Out**: Steps out of the currently running function. Use this command to quickly get back to the calling function, or end execution of the transaction if this was the entry point of the transaction.
+- **Reset**: Reset the debugger to the beginning of the transaction. Note this will be inactive until you step in some way.
+
+There are 4 panels:
+
+- **Code**: Contains the smart contract code we’re stepping through. Each contract involved in this transaction is displayed in a tab. If we’re unable to get the source code of a contract, we can still debug the transaction, but that particular contract will be opaque and you’ll see an “Unknown Contract” tab noting this.
```

Comment:
Should we call this section "Source Code"? Another small thing but do whatever you want about this